### DISCUSSION TITLE
Quick selection in tray icon

### DISCUSSION THREAD

**Discussion by JahnSteven** [snappysnappydog/Simple-IP-Config#119] [Ideas]
My job requires me to visit multiple sites daily and I'm constantly switching static IP addresses between sites and then back to DHCP when I'm in the office. Could a feature be added upon right clicking the tray that it displays the last 5 or so recently used IP settings from a single adapter.

**Comment by blackcrack** [comment]
you could make a patch/pullrequest.. if you in able to programming this, Kurtis could add this..
and... a possible to add/change the Workgroups could also be cool :)

**Comment by KurtisLiggett** [comment]
Seems like a useful feature that I've thought about before. I have some other things I'd like to finish up before this though. Any help would be welcomed.

**Comment by cruzer619** [comment]
I know this is old..  just giving my +1 would be nice to have this feature..  thanks for the great app..
